### ISSUE TITLE
Some queries are really just testing parquet

### ISSUE THREAD

**Issue by mrocklin** [coiled/coiled-runtime#493]
I was looking at some of the H2O queries and, the way that they're set up, they're like 90% just testing the performance of read_parquet.  It seems like we should either remove parquet from what is measured (by a persist and wait for example) or stop caring about these queries, and maybe just care about one of them.

Alternatively, was this known/intentional?

**Comment by hayesgb** [comment]
It's known.

As-is they show end-to-end performance from the POV of the user.  This surfaces things like the impact of  implementing column projection, string dtypes, and https://github.com/dask/dask/issues/9619.

**Comment by rjzamora** [comment]
In an ideal world, we could record and analyze both the end-to-end and io-free run times seperately. However, the persist and wait approach is not always possible for large-scale datasets.

**Comment by mrocklin** [comment]
It is possible here though right?  These datasets are small.  I'm totally fine if we want to do an end-to-end workflow, but currently it looks like we have several end-to-end workflows that are all testing 95% the same thing.  Maybe this is just an artifact of H2O and we're fine doing a weird job here because we're trying to match some existing standard.  I do want to make sure that we're on the same page though that lots of these queries are super-correlated.  If we're not aware of that then I recommend that we spend more time looking at profiles rather than just taking a birds-eye view.

**Comment by hayesgb** [comment]
Yes.  Given that all the queries now run reliably now on a 50GB dataset, I'm in favor of updating the test suite, so we could include this information as well  For the sake of completeness, the following table show the fraction of time spent in `_read_table_from_path` for each query on both a 5GB and 50GB dataset.


<google-sheets-html-origin style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">

FS | Dataset Size (GiB) |   | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
  |   | 0 |   |   |   |   |   |   |   |   |  
s3fs | 5 |   | 92 | 74 | 49 | 67 | 51 | 46 | 44 | 4 | 37
  | 50 |   | 92 | 84 | 50 | 73 | 54 | 54 | 50 | 10 | 52

</google-sheets-html-origin>


This table shows the total Wall Clock time (seconds) for each query:

<google-sheets-html-origin style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">

FS | Dataset Size (GiB) |   | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
  |   | 0 |   |   |   |   |   |   |   |   |  
s3fs | 5 |   | 4.7 | 3.4 | 26 | 11 | 16 | 16 | 23 | 155 | 23
  | 50 |   | 36 | 41 | 622 | 199 | 357 | 245 | 589 | 1655 | 311

</google-sheets-html-origin>
